source/css/_common/components/components.styl

```diff
@@ -3,7 +3,6 @@
 
 @import "buttons";
 @import "pagination";
-@import "comments";
```

Comment:
This changes are relevant to the fonts?

Reply:
Yes, because all content in `comments.styl` is no longer needed, when `font-size` is no longer hardcoded.

Reply:
Screens with comments?

Reply:
Style of comments not changed